modules/scan/admin-bar-notice.js

```diff
@@ -0,0 +1,50 @@
+( function( localized ) {
+	function ready( fn ) {
+		if ( document.readyState != 'loading' ) {
+			fn();
+		} else {
+			document.addEventListener( 'DOMContentLoaded', fn );
+		}
+	}
+
+	function fetch_scan_treats_and_add_link() {
+		fetch( localized.scan_endpoint, {
```

Comment:
Just to confirm, `fetch` is available in all browsers we support? Or is this script compiled with webpack?

Reply:
I looked into this, and we have as a dependency a fetch polyfill package called `unfetch`.